File: 프로그래머스/lv2/70129. 이진 변환 반복하기/이진 변환 반복하기.py
def solution(s):
    countZero = 0
    count = 0
    while s != "1":
        countZero+= s.count('0')
        count+=1
        s=s.replace('0','')
        s=bin(len(s))[2:]
    return [count, countZero]

File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.js
function solution(s) {
    let zeroCount = 0
    let transCount = 0
    
    while(s !== '1') {
        const count = s.split('').filter((char) => char === '0').length
        
        s = s.replaceAll('0', '').length.toString(2)
        transCount+=1
        zeroCount+=count
    }
    
    return [transCount, zeroCount]
    
}